### ISSUE TITLE
Regression: Networking NexusWebApi Tests Db is null failures

### ISSUE THREAD

**Issue by Al12rs** [Nexus-Mods/NexusMods.App#1326]
Networking NexusWebApi tests fail with nullpointer exception on _conn.Db being null in the `GameRegistry.StartupAsync` method.

```
System.NullReferenceException: Object reference not set to an instance of an object.

System.NullReferenceException
Object reference not set to an instance of an object.
   at NexusMods.DataModel.GameRegistry.GameMetadata.All(IDb db) in C:\Dev\Repos\NexusMods.App\src\NexusMods.DataModel\GameRegistry\GameMetadata.cs:line 29
   at NexusMods.DataModel.Registry.Startup(IEnumerable`1 games) in C:\Dev\Repos\NexusMods.App\src\NexusMods.DataModel\GameRegistry\Registry.cs:line 59
   at NexusMods.DataModel.Registry.StartAsync(CancellationToken cancellationToken) in C:\Dev\Repos\NexusMods.App\src\NexusMods.DataModel\GameRegistry\Registry.cs:line 123
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__15_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>g__LogAndRethrow|15_3(<>c__DisplayClass15_0&)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Xunit.DependencyInjection.DependencyInjectionTestFrameworkExecutor.RunTestCases(IEnumerable`1 testCases, IMessageSink executionMessageSink, ITestFrameworkExecutionOptions executionOptions)

   ```

**Comment by halgari** [comment]
So this is often caused by something being broken deep in the internals of Mnemonic DB, often when a new attribute is added. I'll try to find a btter way of surfacing this error

**Comment by halgari** [comment]
This should be fixed or drastically reduced by #1330

**Comment by Al12rs** [comment]
Can't reproduce anymore
